kaldb/src/main/java/com/slack/kaldb/logstore/opensearch/OpenSearchAdapter.java

```diff
@@ -142,13 +144,16 @@ public Query buildQuery(
           && !queryStr.equals("*")) {
         QueryStringQueryBuilder queryStringQueryBuilder = new QueryStringQueryBuilder(queryStr);
 
-        // the field to use when none are provided, otherwise will OR all fields in the registry (up
-        // to a configurable field limit)
-        queryStringQueryBuilder.defaultField("_all");
+        if (queryShardContext.getMapperService().fieldType("_all") != null) {
+          queryStringQueryBuilder.defaultField("_all");
```

Comment:
Do we have a static somewhere that references `_all`? I know it wasn't that way in the existing code, but wondering about it now. 🤔